apps/dialtone-documentation/docs/about/whats-new/posts/2024-4-11.md

```diff
@@ -0,0 +1,282 @@
+dist/lib/ivr-node.cjs                              4.48 kB │ gzip: 1.83 kB │ map:  9.43 kB
+dist/lib/collapsible.cjs                           4.52 kB │ gzip: 1.70 kB │ map: 12.32 kB
+dist/lib/utils.cjs                                 4.56 kB │ gzip: 2.19 kB │ map: 17.15 kB
+dist/chunks/tab-7hJQSLFx.js                        5.36 kB │ gzip: 1.86 kB │ map: 13.03 kB
+dist/chunks/dropdown-zhMEz3bn.js                   5.53 kB │ gzip: 2.07 kB │ map: 16.79 kB
+dist/lib/modal.cjs                                 5.66 kB │ gzip: 2.10 kB │ map: 15.31 kB
+dist/lib/tooltip.cjs                               5.74 kB │ gzip: 2.25 kB │ map: 19.16 kB
+dist/lib/avatar.cjs                                5.87 kB │ gzip: 2.20 kB │ map: 19.60 kB
+dist/lib/attachment-carousel.cjs                   5.90 kB │ gzip: 1.84 kB │ map: 13.76 kB
+dist/lib/combobox-with-popover.cjs                 6.21 kB │ gzip: 2.35 kB │ map: 15.21 kB
+dist/chunks/index-2jPosQBn.js                      6.81 kB │ gzip: 2.26 kB │ map: 15.96 kB
+dist/lib/skeleton.cjs                              7.24 kB │ gzip: 2.10 kB │ map: 23.48 kB
+dist/lib/general-row.cjs                           8.41 kB │ gzip: 2.87 kB │ map: 18.36 kB
+dist/lib/input.cjs                                 8.62 kB │ gzip: 2.62 kB │ map: 22.12 kB
+dist/lib/combobox-multi-select.cjs                 9.04 kB │ gzip: 3.07 kB │ map: 22.41 kB
+dist/lib/message-input.cjs                        11.55 kB │ gzip: 3.59 kB │ map: 22.70 kB
+dist/lib/datepicker.cjs                           13.13 kB │ gzip: 3.47 kB │ map: 34.18 kB
+dist/lib/rich-text-editor.cjs                     14.56 kB │ gzip: 5.17 kB │ map: 52.99 kB
+dist/dialtone-vue.cjs                             14.98 kB │ gzip: 3.94 kB │ map:  0.10 kB
+dist/lib/editor.cjs                               14.98 kB │ gzip: 4.15 kB │ map: 28.14 kB
+dist/lib/popover.cjs                              14.98 kB │ gzip: 4.58 kB │ map: 41.03 kB
+dist/lib/emoji-picker.cjs                         23.10 kB │ gzip: 6.21 kB │ map: 50.83 kB
+✓ built in 2.15s
+```
+
+## Migration to dialtone >=9.27.1
+
+Replace any instances of the old imports with the corresponding new imports.
+Refer to the examples table below for clarity.
+
+| Previous 👎                                                                      | New 👍                                                                          | Enable Tree shaking (best) ✨                                                        |
+|----------------------------------------------------------------------------------|---------------------------------------------------------------------------------|-------------------------------------------------------------------------------------|
+| import { DtEmoji } from '@dialpad/dialtone/vue/emoji'                            | import { DtEmoji } from '@dialpad/dialtone/vue'                                 | import { DtEmoji } from '@dialpad/dialtone/vue/lib/emoji.js'                        |
```

Comment:
Can we make these code? not sure if triple ` works within a table

Reply:
Triple backticks doesn't work within a table, I can add single backticks but I don't like too much how it looks.

![Screenshot 2024-04-15 at 4 41 49 p m](https://github.com/dialpad/dialtone/assets/87546543/e64c7410-acae-4a83-9f33-671e48405003)

Reply:
I think this way is slightly better. I'll let you decide what you think looks best.

Reply:
Yeah, definitely looks better on the website, I was looking on the markdown preview.